pkg/controller/siddhiprocess/config.go

```diff
@@ -178,6 +180,8 @@ type Configs struct {
 	NATSTimeout        int
 	DefaultRTime       int
 	DeploymentSize     int32
+	MaxUnavailable     int32
```

Comment:
Do we need this as int32 because of the K8s objects?

Reply:
Since K8s structs that use this parameter need `int32`. If we use int we need to execute an unnecessary conversion step.